### PR TITLE
Added additional cypress tests for Security Analytics Dashboards plugin.

### PR DESCRIPTION
**Pull request by AWSHurneyt**
### Description
1. Fixed the exiting cypress tests to accommodate new functionality.
1. Added additional cypress tests and related commands.

### Issues Resolved

[List any issues this PR will resolve]

### Check List

- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by AWSHurneyt** [comment]
Still investigating the Security Analytics cypress test failures. They pass when run in a local functional test repo package, but are failing when run as part of a github action.

**Comment by AWSHurneyt** [comment]
The security analytics cypress tests are succeeding now. There is 1 failing ISM cypress test, but that shouldn't be related to the security analytics tests. Will look into resolving that test.

**Comment by AWSHurneyt** [comment]
I was able to reproduce the failing ISM test locally, and it does **not** seem to be an issue with the tests added by the addition of the security analytics tests.

The test is failing because the `admin` user doesn't have the permissions necessary to delete aliases. That permissions issue is causing the test to stall and fail because a sample alias needs to be deleted for the test to proceed. It appears the cypress tests were failing in PR https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/446 which merged these ISM tests to the 2.5 branch. @SuZhou-Joe, did you notice if the test failed for a similar reason in your initial PR?

I manually tested this permissions issue using the `docker-compose.yml` procedure described [here](https://github.com/opensearch-project/opensearch-build/issues/2908#issuecomment-1396344843) to bring up a security-enabled domain. I logged into the domain using the `admin/admin` credentials, created an alias using the ISM UI, and attempted to delete that alias using the UI. The error popup below was thrown.

Screen Shot 2023-02-09 at 4 56 25 PM

I then attempted to delete the alias using devtools, and received the same error.
```
{
  "error": {
    "root_cause": [
      {
        "type": "security_exception",
        "reason": "no permissions for [] and User [name=admin, backend_roles=[admin], requestedTenant=__user__]"
      }
    ],
    "type": "security_exception",
    "reason": "no permissions for [] and User [name=admin, backend_roles=[admin], requestedTenant=__user__]"
  },
  "status": 403
}
```

To reiterate, the failing ISM cypress test is not related to the additional security analytics tests added in this PR.

**Comment by CCongWang** [comment]
hi @AWSHurneyt , I see destination branch is 2.5, any specific reason that you don't want to merge to main? If it's a change that can be used in all versions >=  2.5, usually people merge their change to main, and backport to 2.x and 2.5

File: cypress/utils/plugins/alerting-dashboards-plugin/commands.js [hunk 1]
```diff
@@ -121,7 +121,7 @@ Cypress.Commands.add(
   'insertDocumentToIndex',
   (indexName, documentId, documentBody) => {
     cy.request(
-      'PUT',
+      'POST',
```
[the remark this answers is not in the document]
Comment (AWSHurneyt) [reply cypress/utils/plugins/alerting-dashboards-plugin/commands.js:124]: Yes, that's intentional. I received an error message about the method used by that command. The message stated the `PUT` method wasn't valid for that request.

File: cypress/integration/plugins/index-management-dashboards-plugin/aliases.js [hunk 2]
```diff
@@ -118,6 +118,7 @@ describe('Aliases', () => {
       cy.get('[data-test-subj="deleteConfirmButton"]').should(
         'not.be.disabled'
       );
+      cy.wait(5000);
```
[the remark this answers is not in the document]
Comment (AWSHurneyt) [reply cypress/integration/plugins/index-management-dashboards-plugin/aliases.js:121]: Thank you for catching this! I added this when troubleshooting the ISM test failure. I'll revise this PR to remove this change.

**Comment by AWSHurneyt** [comment]
> hi @AWSHurneyt , I see destination branch is 2.5, any specific reason that you don't want to merge to main? If it's a change that can be used in all versions >= 2.5, usually people merge their change to main, and backport to 2.x and 2.5

@CCongWang 
The security analytics dashboards `main` branch isn't currently set to version 3.0 as the backend plugin is still on version 2.5. I believe the plan is to investigate bumping the plugin to 3.0 after the 2.5 release as the alerting plugin ran into issues when attempting to bump to 3.0. Because there are no 3.0 artifacts for security analytics, I was planning to merge these new tests to the 2.5 branch, and then create a separate PR to add them to the 3.0 branch once the plugin is successfully bumped.

**Review by CCongWang — APPROVED** [review]

**Review by tianleh — APPROVED** [review]

**Pull request merged**